packages/kbn-securitysolution-list-hooks/src/use_persist_exception_item/index.ts

```diff
@@ -47,57 +47,63 @@ export const usePersistExceptionItem = ({
   const [isLoading, setIsLoading] = useState(false);
   const isUpdateExceptionItem = (item: unknown): item is UpdateExceptionListItemSchema =>
     Boolean(item && (item as UpdateExceptionListItemSchema).id != null);
+  const isSubscribed = useRef(false);
 
   useEffect(() => {
-    let isSubscribed = true;
-    const abortCtrl = new AbortController();
+    let abortCtrl: AbortController | null = null;
+    isSubscribed.current = true;
     setIsSaved(false);
 
     const saveExceptionItem = async (): Promise<void> => {
-      if (exceptionListItem != null) {
-        try {
-          setIsLoading(true);
-
-          if (isUpdateExceptionItem(exceptionListItem)) {
-            // Please see `x-pack/plugins/lists/public/exceptions/transforms.ts` doc notes
-            // for context around the temporary `id`
-            const transformedList = transformOutput(exceptionListItem);
-
-            await updateExceptionListItem({
-              http,
-              listItem: transformedList,
-              signal: abortCtrl.signal,
-            });
-          } else {
-            // Please see `x-pack/plugins/lists/public/exceptions/transforms.ts` doc notes
-            // for context around the temporary `id`
-            const transformedList = transformNewItemOutput(exceptionListItem);
-
-            await addExceptionListItem({
-              http,
-              listItem: transformedList,
-              signal: abortCtrl.signal,
-            });
-          }
-
-          if (isSubscribed) {
-            setIsSaved(true);
-          }
-        } catch (error) {
-          if (isSubscribed) {
-            onError(error);
-          }
+      if (exceptionListItem === null) {
+        return;
+      }
+
+      try {
+        abortCtrl = new AbortController();
```

Comment:
won't every new call of `saveExceptionItem` create a new instance of `AbortController`?
That would lead to replacing value in `abortCtrl` and older requests might not be cancelled when component gets unmounted.

Reply:
I tried to test this, but turned out this hook is not used anymore in code.

@eokoneyo, would you mind to delete it and associated test? Thanks

Reply:
> I tried to test this, but turned out this hook is not used anymore in code.
> 
> @eokoneyo, would you mind to delete it and associated test? Thanks

I think this is outside the scope of this PR, I've however created an issue for this here https://github.com/elastic/kibana/issues/201917